### ISSUE TITLE
forward of the model

### ISSUE THREAD

**Issue by cwlinghk** [YoussefAch/RPS-classification-using-PyTorch#1]
In the forward of the model, the last line should be

F.log_softmax(h, dim=1) instead of F.log_softmax(h, dim=0)

if you do it in a batch, am I correct?
